File: ClinicControlCenter/ClientApp/src/libs/util/base-api/src/lib/models/PaginationFilter.ts
export interface IPaginationFilter {
  Limit?: number;
  Offset?: number;

  SortBy?: string;
  Asc?: boolean;
}

export class PaginationFilter implements IPaginationFilter {
  Limit: number = 10;
  Offset: number = 0;

  SortBy?: string;
  Asc?: boolean;
}

File: SDK/Pagination/PaginationFilter.cs
namespace SDK.Pagination
{
    public class PaginationFilter : IPaginationFilter
    {
        public int? Limit { get; set; }

        public long? Offset { get; set; }

        public string SortBy { get; set; }

        public bool? Asc { get; set; }
    }
}
